packages/flame/src/Alert/Alert.tsx

```diff
@@ -56,39 +56,34 @@ export interface AlertProps {
   onClose?: Function;
   /** Whether a Close button appears */
   noCloseBtn?: boolean;
+  /** Icon for the alert */
+  icon?: React.ReactNode;
```

Comment:
@inooid Why a node instead of an icon name? just curious, will we ever need to pass a custom icon?

Reply:
Mostly due to code splitting (same reason why the generic `<Icon />` component suffers from pulling in the entire icon pack), but also because of flexibility. The internal design system docs also show a use-case where they use the present icon and I think there will be more use-cases where some products would like to use a different icon.

![image](https://user-images.githubusercontent.com/1291263/66137209-08490300-e5fd-11e9-952a-37a58969a8be.png)


@maartenafink is setting up a bunch of guidelines and is discussing with the design team whether the alerts should **always** have an icon, but that will be an easy change in the future if that's needed 👍

Reply:
@inooid by name I meant passing any name from the icons that we have, so that will cover the case of the present icon since it's already in the design system. But it makes sense for this 

> (same reason why the generic `<Icon />` component suffers from pulling in the entire icon pack)